### PR TITLE
port defaultLazySymbolResolution, forceLazySymbolResolution into openj9

### PR DESCRIPTION
**Pull request by OscarQQ**
related: backlog/issue 391

Signed-off-by: OscarQQ <Yixin.Qian@ibm.com>

### CODE REVIEW

**Comment by OscarQQ** [comment]
| Build       | Platform           | Java Version  | JDK Impl |
|:-------------:|:-------------:|:-----:|:-----------:|
| Grinder/11321     | ppc64le_linux | 8 | IBM|
| Grinder/11322     | s390x_linux      |   8 | IBM|
| Grinder/11323 | x86-64_linux      |    8 | IBM|
| Grinder/11325 | ppc64le_linux      |    8 | openj9|
| Grinder/11326 | ppc64le_linux      |    11 | openj9|
| Grinder/11327 | s390x_linux     |    8 | openj9|
| Grinder/11328 | s390x_linux | 11 | openj9|
| Grinder/11329     | x86-64_linux      |   8 | openj9|
| Grinder/11330 | x86-64_linux      |   11 | openj9|

**Comment by OscarQQ** [comment]
@llxia

**Comment by llxia** [comment]
According to the original VM_Extended.csv, we should run both tests with `Mode101, Mode601`.

**Comment by OscarQQ** [comment]
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/4219 aarch64 works

File: test/functional/cmdLineTests/forceLazySymbolResolution/build.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+		</subant>
+	</target>
+
+	<target name="dist" depends="buildvm" description="generate the distribution">
+		<copy todir="${DEST}">
+			<fileset dir="${src}" includes="*.xml"/>
+		</copy>
+	</target>
+
+	<target name="build" >
```
Comment (llxia) [inline test/functional/cmdLineTests/forceLazySymbolResolution/build.xml:48]: We need to add a clean target. Similar to other build.xml
In this clean target, we need to call https://github.com/eclipse/openj9/blob/3caadd433c8157b403f337ecd7a057ae188d8297/test/functional/VM_Test/build.xml#L195

**Comment by llxia** [comment]
jenkins test extended jdk8,jdk11

**Comment by llxia** [comment]
jenkins test extended all jdk8,jdk11

**Comment by OscarQQ** [comment]
[TestAttachErrorHandling_0](https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_ppc64_aix_Personal_testList_1/18/console) failure on aix is related to #3081

**Comment by llxia** [comment]
PR build link: https://ci.eclipse.org/openj9/view/Pull%20Requests/job/PullRequest-OpenJ9/4639/

**Comment by llxia** [comment]
jenkins test extended all jdk8,jdk11

**Comment by OscarQQ** [comment]
vmfarm build_id: 457891

**Comment by llxia** [comment]
jenkins test extended all jdk8,jdk11

**Comment by llxia** [comment]
jenkins test extended all jdk8,jdk11

**Comment by llxia** [comment]
jenkins test extended xlinuxlargeheap jdk11

**Comment by llxia** [comment]
jenkins test extended plinuxlargeheap jdk11

**Comment by OscarQQ** [comment]
vmfarm build id = 458298

**Comment by OscarQQ** [comment]
vmfarm build id 458484

**Review by llxia — APPROVED** [review]
LGTM

**Pull request merged**